modules/backend/widgets/form/partials/_field_dropdown.php

```diff
@@ -5,35 +5,29 @@
 ?>
 
 <!-- Dropdown -->
-<?php if ($this->previewMode || $field->readOnly): ?>
-    <div class="form-control" <?= $field->readOnly ? 'disabled="disabled"' : ''; ?>>
-        <?= (isset($fieldOptions[$field->value])) ? e(trans($fieldOptions[$field->value])) : '' ?>
-    </div>
-    <input type="hidden" name="<?= $field->getName() ?>" value="<?= $field->value ?>">
```

Comment:
Why did this file need changes? It already had a hidden element.

Reply:
Because this PR tries to stay as close as possible to the regular markup when viewed in previewMode. Unless the markup I removed causes issue (like it did for the checkbox field), I think it's better to stay lean (easier to maintain)

Reply:
Arguably the easiest to maintain is the unchanged code 😜 What's the visual difference between the before / after?

Reply:
> Arguably the easiest to maintain is the unchanged code stuck_out_tongue_winking_eye

Yes, I should have said "future" maintainability...

> What's the visual difference between the before / after?

The after shows a disabled select dropdown with the current value selected.